Content.Server/DropPod/DropPodBeaconComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
```

Comment:
why?

Reply:
They would do imports before building as these are the sourcegenned ones, I would bet.